start.py

```diff
@@ -44,5 +44,11 @@ def start_jenkins():
 
 
 if __name__ == "__main__":
-    proc = start_jenkins()
-    sys.exit(proc.wait())
+    proc = None
+
+    try:
+        proc = start_jenkins()
```

Comment:
Keep in mind that the download of the war file can also take some seconds. If you hit ctrl-c during that time we won't be inside the try block.